libs/dao-data/test/profile.spec.ts

```diff
@@ -22,4 +22,12 @@ describe('haus', () => {
     );
     expect(profile.ens).toBe('samkuhlmann.eth');
```

Comment:
I wonder if we should try to set up a profile for a DAOhaus ENS or something so we don't have to use Sam's in the future?

Reply:
Yes, agreed